openhtf/core/station_api.py

```diff
@@ -213,7 +221,7 @@ class RemoteTest(mutablerecords.Record('RemoteTest', [
     # Timestamps (in milliseconds) that we care about.
     'created_time_millis', 'last_run_time_millis'], {
         # Cache last known state (a RemoteState) so we can detect deltas.
-        'cached_state': None})):
+        'cached_state': None, 'cached_phase_descriptors': None})):
```

Comment:
optional: If you set it to `'cached_phase_descriptiors': list` then the type is documented. An extra empty list is 'created' is the only downside I see